java/ql/src/experimental/Security/CWE/CWE-094/SpringViewManipulation.qhelp

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+
+  <overview>
```

Comment:
If you're just including this as the help for the implicit version of the query, you should mention both queries and the difference between them here

Reply:
Done!